deployment/data-streams/changeset/verifier/v0_5_0/verifier.go

```diff
@@ -0,0 +1,89 @@
+package v0_5_0
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	goEthTypes "github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/types"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/utils/txutil"
+	verifier "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/llo-feeds/generated/verifier_v0_5_0"
+)
+
+type VerifierState struct {
+	Verifier *verifier.Verifier
+}
+
+type HasVerifierAddress interface {
+	GetVerifierAddress() common.Address
+}
+
+// VerifierMethod is a function that, given a loaded VerifierState and config data,
+// returns the EVM tx.
+// T is the type of the config that holds the arguments you need to call the contract.
+type VerifierMethod[T any] func(*VerifierState, T) (*goEthTypes.Transaction, error)
+
+// GetTxs will:
+// iterate over chainSelector => array of T
+// load verifier
+// call the given method
+// wrap in PreparedTx
+func GetTxs[T HasVerifierAddress](
+	e deployment.Environment,
+	contractType string,
+	configsByChain map[uint64][]T,
+	method func(*VerifierState, T) (*goEthTypes.Transaction, error),
+) ([]*txutil.PreparedTx, error) {
+	var preparedTxs []*txutil.PreparedTx
+	for chainSelector, cfgs := range configsByChain {
+		for _, c := range cfgs {
+			contractAddr := c.GetVerifierAddress().Hex()
+
+			state, err := maybeLoadVerifier(e, chainSelector, contractAddr)
```

Comment:
if you make this a `loadState` function - we could put this in the general utils and share across other contract changesets!

Reply:
Good idea!